IDisposableAnalyzers/build/IDisposableAnalyzers.targets

```diff
@@ -0,0 +1,13 @@
+<Project>
+
+  <PropertyGroup>
+    <OwnershipAttributesPath Condition="'$(OwnershipAttributesPath)' == ''">$(MSBuildThisFileDirectory)OwnershipAttributes$(DefaultLanguageSourceExtension)</OwnershipAttributesPath>
+
+    <IncludeOwnershipAttributes Condition="'$(IncludeOwnershipAttributes)' == '' and Exists($(OwnershipAttributesPath))">true</IncludeOwnershipAttributes>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(IncludeOwnershipAttributes)' == 'true'">
+    <Compile Include="$(OwnershipAttributesPath)" Visible="false" />
```

Comment:
Maybe add an error if the OwnershipAttributesPath does not exist?
Even if the build fails because it references a file that doesn't exist, I'm not sure that the message is clear enough.

Reply:
It looks like this currently:

> CSC : error CS2001: Source file 'C:\Users\Joseph\.nuget\packages\idisposableanalyzers\X.Y.Z\build\NullableAttributes.cs' could not be found.

I updated it to look like this:

>C:\Users\Joseph\.nuget\packages\idisposableanalyzers\X.Y.Z\build\IDisposableAnalyzers.targets(20,5): warning IDISP0026: The project specifies <IncludeOwnershipAttributes>true</IncludeOwnershipAttributes>, but including attributes in this language is not yet supported.